quic-client/tests/quic_client.rs

```diff
@@ -80,13 +80,7 @@ mod tests {
             sender,
             exit.clone(),
             staked_nodes,
-            QuicServerParams {
-                max_connections_per_peer: 1,
-                max_staked_connections: 10,
-                max_unstaked_connections: 10,
-                coalesce_channel_size: 100_000, // smaller channel size for faster test
-                ..QuicServerParams::default()
-            },
+            QuicServerParams::default_for_tests(),
```

Comment:
With the new common function, `max_staked_connections` and `max_unstaked_connections` now have different values.

I believe the previous values were somewhat arbitrary, but calling it out for completeness